Libraries/LibJS/Runtime/ECMAScriptFunctionObject.cpp

```diff
@@ -250,8 +250,6 @@ ECMAScriptFunctionObject::ECMAScriptFunctionObject(FlyString name, ByteString so
                     // 3. Perform ! env.InitializeBinding(n, undefined).
                     m_var_names_to_initialize_binding.append({
                         .identifier = id,
-                        .parameter_binding = parameter_bindings.contains(id.string()),
-                        .function_name = function_names.contains(id.string()),
```

Comment:
Maybe leave a comment about this, as I can imagine this becoming a footgun down the line